recipe/conda_build_config.yaml

```diff
@@ -1,18 +1,16 @@
-c_stdlib:          # [linux64]
-  - sysroot        # [linux64]
-c_stdlib_version:  # [linux64 or (osx and x86_64)]
-  - 2.17           # [linux64]
+c_stdlib_version:  # [linux or (osx and x86_64)]
+  - 2.17           # [linux]
   - 10.15          # [osx and x86_64]
 
 MACOSX_SDK_VERSION:         # [osx]
   - 12.3                    # [osx]
 
 blas_impl:
-  - mkl                        # [x86 or x86_64]
+  - mkl                         # [x86_64]
   - generic
 
 github_actions_labels:
-- cirun-openstack-gpu-large    # [x86 or x86_64]
+- cirun-openstack-gpu-2xlarge   # [x86_64]
 - cirun-openstack-cpu-large
```

Comment:
One of the big issues here with the cirun sizes is that we need to distinguish CUDA builds from non-CUDA builds, but that's really hard to do in a local pinning, which is I guess why the distinction on `x86_64` through `blas_impl` is kind of a work-around. But it also means we cannot set the right CI runner for aarch+CUDA.

I've tried to come up with a solution for this, but it would need changes to the global pinning: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5903